src/map.config.json

```diff
@@ -215,6 +215,12 @@
         "altitudeMode": "relativeToGround",
         "heightAboveGround": 5
       }
+    },
+    {
+      "name": "customLayer",
```

Comment:
Could you use a name that explains the layer more? Perhaps we should list this layer also on the `RENNES_LAYER` in `src/stores/layers.ts`